freeipa/src/main/java/com/sequenceiq/freeipa/controller/FreeIpaV1Controller.java

```diff
@@ -86,6 +96,23 @@ public DescribeFreeIpaResponse create(@Valid CreateFreeIpaRequest request) {
         return freeIpaCreationService.launchFreeIpa(request, accountId);
     }
 
+    @Override
+    public void registerChildEnvironment(@Valid RegisterChildEnvironmentRequest request) {
+        ValidationResult validationResult = registerChildEnvironmentRequestValidator.validate(request);
+        if (validationResult.hasError()) {
+            LOGGER.debug("FreeIPA request has validation error(s): {}.", validationResult.getFormattedErrors());
```

Comment:
not `FreeIPA request`, it's `RegisterChildEnvironmentRequest`

Reply:
Fixed